static/variables.less

```diff
@@ -27,3 +27,9 @@
 
 /*----- OTHER -----*/
 @backdrop-blur: saturate(180%) blur(20px);
+
+/*----- Z-INDEX -----*/
+@z-index-1: 1;
+@z-index-minus-1: -1;
+@z-index-big; 99;
+@z-index-really-big: 999999;
```

Comment:
Did you consider giving these semantic names (e.g. `z-index-over`, `z-index-under`, `z-index-dropdown, `z-index-header`)?

Also, `@z-index-big; 99;` has a syntax error… is it even needed?